simulate-travis.py

```diff
@@ -59,6 +59,42 @@
 HERE = os.path.abspath(os.path.dirname(__file__))
 
 
+def _ensure_build_dir_not_empty():
+    """Make sure the build directory can serve as a valid channel.
+    If the directory is empty or contains no builds, the update
+    command will fail. In this case, a dummy package is installed to
+    to create conda index in the build directory.
+    """
+    import bioconda_utils.docker_utils
+    build_dir = bioconda_utils.docker_utils.get_host_conda_bld()
+
+    ADD_CHANNEL_CMD = [
+        'conda', 'config', '--add', 'channels', 'file://%s' % build_dir]
+    REMOVE_CHANNEL_CMD = [
+        'conda', 'config', '--remove', 'channels', 'file://%s' % build_dir]
+    CONDA_UPDATE_DRY_RUN_CMD = ['conda', 'update', '--dry-run', '--all']
+
+    with open(os.devnull, 'w') as DNULL:
```

Comment:
It would be better to report stderr/stdout for easier debugging. Maybe preface it with a log statement saying what's being done.